main/version.h

```diff
@@ -0,0 +1,11 @@
+#ifndef VERSION_H
+#define VERSION_H
+
+#include <stdint.h>
+
+const char *version = "0.9.0";
+const uint16_t version_major = 0;
```

Comment:
Any format is fine, although 5 bits per field fits into a single uint16_t.

Reply:
I'd like to keep it incredibly simple. 3 ints makes sense to me